time stamp
float32 px
float32 py
float32 lonvel
float32 lonacc
float32 yaw
float32 phi
float32 omega
float32 tkerror
float32 tkyaw
float32 tklon
float32 tklat
